grakn-graph/src/main/java/ai/grakn/graph/internal/GraknComputerImpl.java

```diff
@@ -54,7 +54,16 @@ public ComputerResult compute(MapReduce mapReduce) {
         try {
             return getComputer().mapReduce(mapReduce).submit().get();
         } catch (InterruptedException | ExecutionException e) {
-            throw new RuntimeException(e);
+            throw asRuntimeException(e);
+        }
+    }
+
+    private RuntimeException asRuntimeException(Throwable throwable) {
```

Comment:
@fppt Graql shell does not show runtime exceptions. If a vertex program throws IllegalStateException, it would be wrapped as a runtime exception, but we do wanna show our user the error messages.